Automated Test Cases/Automated Test Cases/Test Entities/AutomationPracticeTests.cs

```diff
@@ -0,0 +1,187 @@
+using Automated_Test_Cases.Constant_Entities;
+using Automated_Test_Cases.Test_Entities.BaseTest_Entity;
+using NUnit.Framework;
+using NUnit.Allure.Attributes;
+using Allure.Commons;
+using NUnit.Allure.Core;
+
+namespace Automated_Test_Cases
+{
+    [TestFixture]
+    [AllureNUnit]
+    [AllureSuite("Automation Practices Tests")]
+    [AllureDisplayIgnored]
+    public class AutomationPracticeTests : BaseTest
+    {
+        /*
+         *************************************** AP-1 Verify the ability to create an account  *****************************************************
+         */
+
+        [Test(Description = "Verify the ability to create an account")]
+        [AllureTag("CI")]
+        [AllureSeverity(SeverityLevel.critical)]
+        [AllureTms("AP-1")]
+        [AllureOwner("Alexander Ulyanitskiy")]
+        [AllureSubSuite("Verification")]
+        [Order(1)]
+        public void VerifyTheAbilityToCreateAnAccount()
+        {
+            automationPracticeHomePage?
+                .GoTo(Constant.Url.Automation_Practice_Home_Page_Url);
+            Assert.IsTrue(automationPracticeHomePage?.
+                AtPageByURL(Constant.Url.Automation_Practice_Home_Page_Url), $"Home Page Url is incorrect - test received {driver?.Url}");
+            Assert.IsTrue(automationPracticeHomePage?.
+                AtPageByTitle(Constant.Title.Automation_Practice_Home_Page_Title), $"Home Page title is incorrect - test received {driver?.Title}");
+            automationPracticeHomePage?.Register(Constant.Credentials.Email);
+
+            Assert.IsTrue(automationPracticeLoginPage?.IsDisplayedYourPersonalInformationBlock(), "Personal Information block isn't displayed");
+            Assert.IsTrue(automationPracticeLoginPage?.IsDisplayedYourAddressBlock(), "Your address block isn't displayed");
+            automationPracticeLoginPage?
+                .FirstNameInitialize(Constant.Credentials.FirstName)
```

Comment:
Wrap all these methods into one general

Reply:
Reworked test methods. They became a part of one big method.